features/step_definitions/cli_steps.js

```diff
@@ -12,7 +17,7 @@ export default function cliSteps () {
   mkdirp(tmpDir)
 
   function uniqueId (prefix) {
-    const id = `${++idCounter}`
+    const id = `${++idCounter}` // eslint-disable-line no-plusplus
```

Comment:
It's actually different `++idCounter` !== `idCounter += 1`, but will trying to see if I can get something close